controller/server/controller.go

```diff
@@ -203,42 +193,21 @@ func (c *controller) createServiceInstance(in *scmodel.ServiceInstance) error {
 		params["bindings"] = fromBindings
 	}
 
-	// Then actually make the request to reify the service instance
-	createReq := &sbmodel.ServiceInstanceRequest{
-		ServiceID:  in.ServiceID,
-		PlanID:     in.PlanID,
-		Parameters: params,
-	}
-
-	jsonBytes, err := json.Marshal(createReq)
-	if err != nil {
-		return err
-	}
-
+	// Get broker for this service class.
 	broker, err := c.getBroker(in.ServiceID)
 	if err != nil {
 		return err
 	}
+	cli := util.CreateCFV2BrokerClient(broker)
```

Comment:
I totally misread 'cli' here as 'CLI'.  I have a strong brain-muscle-memory for `client` as a variable name.  Thoughts?